src/components/GetApi/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from "react";
+import axios from "axios";
+
+const Index: React.FC = () => {
+  useEffect(() => {
+    weatherData();
+  }, []);
+
+  const weatherData = async (): Promise<void> => {
+    try {
+      let response = await axios.get(
+        `https://api.openweathermap.org/data/2.5/weather?q=New York&appid=5a6a3604d50f420072819316b2ed23a2`
+      )
+      console.log(response)
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  return <div></div>;
```

Comment:
WHat's the use of the jsx component here?

Reply:
I will remove it